src/clients/rust/src/conversions.rs

```diff
@@ -1,13 +1,13 @@
 pub use super::*;
 
 #[rustfmt::skip]
-impl From<tbc::tb_create_accounts_result_t> for CreateAccountResult {
-    fn from(other: tbc::tb_create_accounts_result_t) -> CreateAccountResult {
+impl From<u32> for CreateAccountResult {
```

Comment:
I think we should constrain `CreateAccountResult` and other enums to only include known variants. 

We do pretty strict version checks server-side, so if we ever materialize a code that's not known to the client, that'll be a pretty big bug for us (if an old client connects to the new cluster, we'll take care to only send old codes to it). But that's not really related to the PR at hand!

Reply:
I've removed the `Unknown` variants and changed the conversions to panics.